plugins/percol/percol.plugin.zsh

```diff
@@ -0,0 +1,22 @@
+if which percol &> /dev/null; then
+    function percol_select_history() {
+        local tac
+        tac="tail -r"
+        BUFFER=$(fc -l -n 1 | eval $tac | percol --query "$LBUFFER")
+        CURSOR=$#BUFFER
+        zle -R -c
+    }
+
+    zle -N percol_select_history
+    bindkey '^R' percol_select_history
+fi
+
+if which marks &> /dev/null; then
```

Comment:
Should this test be nested inside the `if which percol` test above, since it also depends on `percol` being installed?